NitroxClient/Unity/Helper/DebugUtils.cs

```diff
@@ -7,6 +7,46 @@ namespace NitroxClient.Unity.Helper
 {
     public class DebugUtils
     {
+        private static string prefix = "";
+
+        public static void DumpHierarchy(GameObject obj)
+        {
+            Log.Debug("Analyze gameobject: ");
+            Log.Debug("------------------------------------");
+            StartHierachy(obj);
+            Log.Debug("------------------------------------");
+        }
+
+        private static void StartHierachy(GameObject obj)
```

Comment:
Are you using this typo intentionally to distinguish `StartHierarchy` from `StartHierachy`?